### DISCUSSION TITLE
Quickfix does not add import to the existing line

### DISCUSSION THREAD

**Discussion by EN20M** [microsoft/pylance-release#3458] [Enhancement]
Type: <b>Bug</b>

# Behaviour
## Expected vs. Actual

When having an import error handling guard like this:
```
try:
    import foo
    import bar as b
    from ctypes import c_int16, c_uint8
except ModuleNotFoundError as err:
    print(err)
    input('Press ENTER to leave')
```
When using the quickfix for adding a missing ctypes import it does not get added to the existing line but rather a new line with:
```
from ctypes import c_void_p
```
when the already existing line is not placed in an import guard the missing one gets inserted into the existing line.

## Steps to reproduce:

1. put this code in a file:
```
try:
    import sys
    import numpy as np
    from ctypes import c_uint8, c_uint16
except ModuleNotFoundError as err:
    print(err)
    input('Press ENTER to leave')

var = c_void_p
```
2. use quickfix to add the missing import c_void_p

# Diagnostic data

-   Python version (& distribution if applicable, e.g. Anaconda): 3.9.13
-   Type of virtual environment used (e.g. conda, venv, virtualenv, etc.): Global
-   Value of the `python.languageServer` setting: Default

<details>

<summary>Output for <code>Python</code> in the <code>Output</code> panel (<code>View</code>→<code>Output</code>, change the drop-down the upper-right of the <code>Output</code> panel to <code>Python</code>)
</summary>

<p>

```
XXX
```

</p>
</details>

<details>

<summary>User Settings</summary>

<p>

```

languageServer: "Pylance"

```

</p>
</details>

Extension version: 2022.16.0
VS Code version: Code 1.71.2 (74b1f979648cc44d385a2286793c226e611f59e7, 2022-09-14T21:03:37.738Z)
OS version: Windows_NT x64 10.0.19044
Modes:
Sandboxed: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10870H CPU @ 2.20GHz (16 x 2208)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|15.79GB (7.90GB free)|
|Process Argv|--crash-reporter-id aa18c06a-65fb-45ca-83a0-f4dbf580f6e2|
|Screen Reader|no|
|VM|0%|
</details>

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
